content/en/getting_started/synthetics/_index.md

```diff
@@ -0,0 +1,215 @@
+## Overview
```

Comment:
Overall: thanks a lot for the great work here, my only concern is that a lot of the content is similar to the already existing doc, and I want to make sure there's added value on this Getting started. Please let me know if my comment does not make sense from a doc POV, but I'm wondering if we could add more specific examples, for instance using a webhook in the message, setting up fast retries to 2, etc.

Reply:
Thanks for the feedback! Incorporated suggestions into the most recent commit. I'm working on adding details about webhook + fast retries. Will let you know when finalized.